src/dataexplorer/dataexplorer-application/src/components/PatientInfo/MyPatients.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Row, Col } from 'antd';
+import Search from 'antd/lib/input/Search';
+import { IHistory } from '../../types/IHistory';
+
+const MyPatients = (props: IHistory) => {
```

Comment:
Same here, if it works it would be nice to just use History directly
```suggestion
const MyPatients = (props: History) => {
```